modules/lang-painless/src/yamlRestTest/resources/rest-api-spec/test/painless/60_script_doc_values_binary.yml

```diff
@@ -20,7 +20,30 @@
             index: test
             id: 1
             body:
-                binary: U29tZSBiaW5hcnkgYmxvYg==
+                binary: "U29tZSBiaW5hcnkgYmxvYg=="
+
+    - do:
```

Comment:
Can you add a test for the iterator?  Just add up the binary as ints or something.

Reply:
Ahh its line 76.